components/sendgrid/sendgrid.app.js

```diff
@@ -12,10 +12,42 @@ module.exports = {
     _apiUrl() {
```

Comment:
I'm really against making methods that do nothing except return an API URL. Can we get rid of these?

They clutter up the app file with a bunch of unnecessary methods. The API URLs aren't going to change that often, and it's easier to identify the API being called if you have the entire URL where the call is being made. I see no net benefit to having them here.

Reply:
Noted. I got rid of the methods returning API URL for Sendgrid actions only. For methods related to Sendgrid sources, they were left unchanged.